src/main/java/jenkins/plugins/slack/pipeline/SlackUploadFileRunner.java

```diff
@@ -77,17 +77,32 @@ public void visit(File file, String relativePath) {
 
     private boolean doIt(File file) {
         try (CloseableHttpClient client = HttpClient.getCloseableHttpClient(proxy)) {
+            String threadTs = null;
+            String theChannels = channels;
+
+            //thread_ts is passed once with roomId: Ex: roomId:threadTs
+            String[] splitThread = channels.split("[:]+");
```

Comment:
```suggestion
            String[] splitThread = channels.split(":", 2);
```

Reply:
I wonder if channels here is an mistake? Perhaps it should be changed to `channel` it is singular is it not?